Geode.Angular/src/app/pages/messages/messages.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import { ColDef } from 'ag-grid-community';
+import { Message } from './message.model';
+import { MessagesService } from './messages.service';
+import { Filter } from '../../shared/models/filter.model';
+
+@Component({
+  selector: 'gd-messages',
+  templateUrl: './messages.component.html',
+  styleUrl: './messages.component.css',
+})
+export class MessagesComponent implements OnInit {
+  properties: string[] = ['Content', 'SentAt'];
+  rowData: Message[] = [];
+  colDefs: ColDef[] = [{ field: 'content' }, { field: 'sentAt' }];
+  isLoading: boolean = false;
+
+  constructor(private messagesService: MessagesService) {}
+
+  ngOnInit(): void {
+    this.updateRowData();
+  }
+
+  updateRowData(filter: Filter | null = null) {
+    this.isLoading = true;
+    this.messagesService
+      .getAllMessages(filter)
+      .subscribe({
+        next: (result) => {
+          this.rowData = result;
```

Comment:
Just write like this: 
`
this.messagesService
      .getAllMessages(filter)
      .subscribe(result => this.rowData = result)
`